simapp/go.mod

```diff
@@ -225,11 +225,14 @@ require (
 
 replace (
 	cosmossdk.io/client/v2 => ../client/v2
+	cosmossdk.io/store => ../store
 	cosmossdk.io/x/circuit => ../x/circuit
 )
 
 // Below are the long-lived replace of the SimApp
 replace (
+	cosmossdk.io/api => ../api
+	cosmossdk.io/log => ../log
```

Comment:
are they required?

Reply:
I don't understand these changes too. Could you explain the replacement.

Reply:
[it](https://github.com/cosmos/cosmos-sdk/blob/fc725a35d6764de62f12ae56e2369d6229aef635/simapp/go.mod#L228) was there before, can be resolve d with rebase